src/deepsparse/server/README.md

```diff
@@ -0,0 +1,120 @@
+### DeepSparse Server 🔌
+
+The DeepSparse inference server allows you to serve models and pipelines in deployment in HTTP. The server runs on top of the popular FastAPI web framework and Uvicorn web server. Currently, the server only supports NLP tasks, however support for computer vision will soon be released in upcoming versions!
```

Comment:
```suggestion
The DeepSparse inference server allows you to serve models and pipelines in deployment in HTTP. The server runs on top of the popular FastAPI web framework and Uvicorn web server. Currently, the server only supports NLP tasks; however, support for computer vision will soon be released in upcoming versions!
```

Reply:
What is Uvicorn? (Is that correct?)

Reply:
it is a web server implementation for Python